src/components/authentication/Login/index.js

```diff
@@ -16,6 +16,9 @@ const Login = () => {
     authState: { token, authError },
   } = useAuth();
 
+  let location = useLocation();
+  let from = location.state?.from?.pathname || "/";
+
```

Comment:
Appreciation: got to learn how to use last location to navigate user after authentication.

Suggestion: Don't use non-generic names instead use name here as lastLocation.